### PR TITLE
Flow for first time login - SSO 

### PR DESCRIPTION
**Pull request by ArifPMan**

### CODE REVIEW

File: v6/enterprise/sso/user_sso.md [hunk 1]
```diff
@@ -23,3 +23,11 @@ After a team admin has enabled single sign-on (**SSO**) for Postman, users can
 Postman redirects the user to the configured Identity Provider. Here, for example, the **G-Suite** Login screen.  
 
 [![G-Suite login](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/59036889.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/59036889.png)  
+
+Once you login after a successful authorization, Postman will take you to its sign in screen again:
+
+[![postma sign-in](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Postman_SignIn_Screen.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Postman_SignIn_Screen.png)
```
Comment (kaustavdm) [inline v6/enterprise/sso/user_sso.md]: `postman`

File: v6/enterprise/sso/user_sso.md [hunk 2]
```diff
@@ -23,3 +23,11 @@ After a team admin has enabled single sign-on (**SSO**) for Postman, users can
...
+
+Enter your Postman credentials again to login to Postman via your security provider. 
+
+**Note:** First, Postman used your credentials to verify account ownership and link the Postman account to the account of your security provider, say for instance, Okta. Postman prompts you to sign in again to actually let you login securely through your security provider.
```
Comment (kaustavdm) [inline v6/enterprise/sso/user_sso.md]: This paragraph reads sketchy. Can you rephrase?

File: v6/enterprise/sso/user_sso.md (hunk 1 above)
Comment (tristandenyer) [reply v6/enterprise/sso/user_sso.md]: @ArifPMan You able to update this? It's getting stale.

File: v6/enterprise/sso/user_sso.md (hunk 2 above)
Comment (ArifPMan) [reply v6/enterprise/sso/user_sso.md]: Addressed your f/b.

**Comment by kaustavdm** [comment]
Closing this PR since this looks stale and we will need to revisit the SSO docs anyway.
